providers/flagd/README.md

```diff
@@ -105,23 +89,28 @@ automatically.
 
 ### Deadline (gRPC call timeout)
 
-The deadline for an individual flag evaluation can be configured by calling `setDeadline(< deadline in millis >)`.
+The deadline for an individual flag evaluation can be configured by calling `setDeadline(myDeadlineMillis)`.
```

Comment:
Why did you make this change?

Reply:
I've never liked the `< xxx >` style. It always looked awkward to me. I'm of the opinion that using an imaginary but well-named var is clearer (usually using `my`).